pkg/exporter/sumologicexporter/README.md

```diff
@@ -44,7 +44,7 @@ exporters:
     trace_format: {otlp}
 
     # timeout is the timeout for every attempt to send data to the backend,
-    # maximum connection timeout is 55s, default = 5s
+    # maximum connection timeout is 55s, default = 30s
```

Comment:
do we think this is too high for the default timeout?

Reply:
We use this in the Helm Chart and it works fine there.